src/lib.rs

```diff
@@ -257,7 +257,8 @@
 //! Note that the CC algorithm should be configured before calling [`connect()`]
 //! or [`accept()`]. Otherwise the connection will use a default CC algorithm.
 //!
-//! [`CongestionControlAlgorithm`]: cc/enum.Algorithm.html
+//! [`CongestionControlAlgorithm`]:
+//! recovery/enum.CongestionControlAlgorithm.html
```

Comment:
This just ends up redirecting to the public alias anyway, so might as well point the docs to that.
```suggestion
//! enum.CongestionControlAlgorithm.html
```